Havit/Havit/Screens/CategoryContents/View/CategoryContentsViewController.swift

```diff
@@ -6,24 +6,184 @@
+        contentsCollectionView.dataSource = self
+        contentsCollectionView.register(ContentsCollectionViewCell_sort2.classForCoder(), forCellWithReuseIdentifier: ContentsCollectionViewCell_sort2.cellID)
+    }
+    
+    func setAutoLayouts() {
+        mainView.snp.makeConstraints { (make) -> Void in
+            make.leading.bottom.trailing.equalToSuperview()
+            make.top.equalTo(view).offset(17)
+        }
+        
+        filterView.snp.makeConstraints { (make) -> Void in
+            make.leading.trailing.equalTo(mainView).offset(0)
+            make.top.equalTo(mainView).offset(17)
+            make.height.equalTo(67)
+        }
+        
+        totalLabel.snp.makeConstraints { (make) -> Void in
+            make.leading.equalTo(filterView).offset(16)
+            make.top.equalTo(filterView).offset(0)
+        }
+        
+        changeShowButton.snp.makeConstraints { (make) -> Void in
+            make.top.equalTo(filterView).offset(0)
+            make.trailing.equalTo(filterView).offset(-16)
+            make.width.equalTo(18)
+            make.height.equalTo(18)
+        }
+        
+        sortButton.snp.makeConstraints { (make) -> Void in
+            make.bottom.equalTo(filterView).offset(-19)
+            make.trailing.equalTo(filterView).offset(-16)
+            make.width.equalTo(47)
+            make.height.equalTo(15)
+        }
+        
+        filterCollectionView.snp.makeConstraints { (make) -> Void in
+            make.leading.equalTo(filterView).offset(0)
+            make.bottom.equalTo(filterView).offset(-9)
+            make.width.equalTo(250)
+            make.height.equalTo(31)
+        }
+        
+        contentsCollectionView.snp.makeConstraints { (make) -> Void in
+            make.top.equalTo(filterView).offset(67)
+            make.leading.equalTo(mainView).offset(0)
+            make.trailing.equalTo(mainView).offset(0)
+            make.bottom.equalTo(mainView).offset(0)
+        }
+    }
+}
+
 
-    /*
-    // MARK: - Navigation
+// MARK: - Extensions
 
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+extension CategoryContentsViewController: UISearchBarDelegate {
+    
+}
+
+extension CategoryContentsViewController: UICollectionViewDelegate {
+    
+}
+
+extension CategoryContentsViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        switch collectionView {
+        case filterCollectionView:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: CategoryFilterCollectionViewCell.cellID, for: indexPath) as! CategoryFilterCollectionViewCell
```

Comment:
P2;
UICollectionView+Extension에 추가되어있는 dequeueReusableCell코드를 사용해서 코드를 짜주시면 감사드리겠습니다~
```suggestion
           let cell: CategoryFilterCollectionViewCell = collectionView.dequeueReusableCell(forIndexPath: indexPath)
```

Reply:
넵 이 코드도 이동하도록 하겠습니다!